pyannote/audio/core/model.py

```diff
@@ -378,6 +378,8 @@ def setup(self, stage=None):
             self.task.setup_validation_metric()
             # this is to make sure introspection is performed here, once and for all
             _ = self.introspection
+            # Put the augmentations on the gpu if we can by registering them on the model
+            self.task._setup_augmentations()
```

Comment:
Why the `_` prefix? Let's be consistent with the other `setup_xxx` methods.

Reply:
Also, I think we should `unregister` the augmentation before checkpoint the model.
Otherwise, there might be problems when loading them back? 

Did you test loading a model that was trained with `in model` transforms?
Does this work?